.github/workflows/main.yml

```diff
@@ -19,7 +19,7 @@ jobs:
   build:
     strategy:
       matrix:
-        os: [ ubuntu-20.04, macos-latest, windows-2019 ]
+        os: [ ubuntu-24.04, macos-latest, windows-2019 ]
```

Comment:
Should we be upgrading to ubuntu-22.04 (oldest compatible Ubuntu) or ubuntu-latest (consistent with macOS and automatically upgraded)?

Reply:
I think, latest should be fine, but it could lead to some issues. I guess we'll see.